manifests/init.pp

```diff
@@ -195,9 +195,9 @@
   $odoo_supported_versions = {
     'Debian' => {
       '9'  => ['11.0', '12.0'],
-      '10' => ['11.0', '12.0', '13.0', '14.0', '15.0', '16.0'],
-      '11' => ['14.0', '15.0', '16.0', 'system'],
-      '12' => ['14.0', '15.0', '16.0', '17.0'],
+      '10' => ['11.0', '12.0', '13.0', '14.0', '15.0', '16.0', '17.0'],
+      '11' => ['14.0', '15.0', '16.0', '17.0'],
```

Comment:
Odoo 17.0 does not work with older versions of Debian, we cannot add it to Debian 10 / 11.  CI shows this failure for Debian 11:

> odoo class when using odoo version="17.0" works idempotently with no errors

Please revert them to fix CI.

(note that the CI does not run against Debian 10 because it has reached EOL, basically we just don't touch anything related to EOL Operating Systems so that users of legacy versions are unaffected by updates, and from time to time we issue a new major version that remove support for them).